packages/expo-dev-menu/android/src/main/java/expo/modules/devmenu/modules/DevMenuPreferences.kt

```diff
@@ -1,24 +1,18 @@
 package expo.modules.devmenu.modules
 
+import android.content.Context
 import android.content.Context.MODE_PRIVATE
 import com.facebook.react.bridge.*
-import com.facebook.react.module.annotations.ReactModule
 import expo.interfaces.devmenu.DevMenuPreferencesInterface
+import expo.modules.kotlin.exception.Exceptions
+import expo.modules.kotlin.modules.Module
+import expo.modules.kotlin.modules.ModuleDefinition
 
 private const val DEV_SETTINGS_PREFERENCES = "expo.modules.devmenu.sharedpreferences"
-private const val DEV_MENU_PREFERENCES = "DevMenuPreferences"
 
-/**
- * Class that represents all user preferences connected with the current [expo.modules.devmenu.interfaces.DevMenuDelegateInterface].
- */
-@ReactModule(name = DEV_MENU_PREFERENCES)
-class DevMenuPreferences(context: ReactApplicationContext) : BaseJavaModule(), DevMenuPreferencesInterface {
-  private val sharedPreferences = context.getSharedPreferences(DEV_SETTINGS_PREFERENCES, MODE_PRIVATE)
-
-  override fun getName() = DEV_MENU_PREFERENCES
-
-  override fun canOverrideExistingModule(): Boolean {
-    return true
+class DevMenuPreferencesHandel(context: Context) : DevMenuPreferencesInterface {
```

Comment:
Did you mean `DevMenuPreferencesHandle`?
```suggestion
class DevMenuPreferencesHandel(context: Context) : DevMenuPreferencesInterface {
```